src/commonMain/kotlin/com/charleskorn/kaml/YamlConfiguration.kt

```diff
@@ -40,7 +40,9 @@ public data class YamlConfiguration constructor(
     internal val polymorphismPropertyName: String = "type",
     internal val encodingIndentationSize: Int = 2,
     internal val breakScalarsAt: Int = 80,
-    internal val sequenceStyle: SequenceStyle = SequenceStyle.Block
+    internal val sequenceStyle: SequenceStyle = SequenceStyle.Block,
+    internal val multiLineStringStyle: MultiLineStringStyle = MultiLineStringStyle.DoubleQuoted,
```

Comment:
Should this default to matching `quotedScalarStyle`? So if the user sets `quotedScalarStyle`, then `mutliLineStringStyle` uses that value unless there's an explicit value given for `multiLineStringStyle`.

```suggestion
    internal val multiLineStringStyle: MultiLineStringStyle = quotedScalarStyle, // This won't compile due to the different types, but hopefully this makes the idea clear.
```